internal/tests/prediction_test.go

```diff
@@ -25,6 +25,8 @@ func TestPredictionSucceeded(t *testing.T) {
 	assert.Equal(t, server.PredictionSucceeded, resp.Status)
 	assert.Equal(t, "*bar*", resp.Output)
 	assert.Equal(t, "starting prediction\nprediction in progress 1/1\ncompleted prediction\n", resp.Logs)
```

Comment:
I think this failed with:
```
expected: "starting prediction\nprediction in progress 1/1\ncompleted prediction\n/home/runner/work/cog-runtime/cog-runtime/.venv-legacy/lib/python3.13/site-packages/cog/server/scope.py:21: ExperimentalFeatureWarning: current_scope is an experimental internal function. It may change or be removed without warning.\nwarnings.warn(\n"
```

So we should check contains instead? Also just realized `actual` & `expected` are flipped but that we can fix in a separate PR.

Reply:
nvm it is `(expected, actual)` I can never remember :brain: